connectivity/check/deployment.go

```diff
@@ -665,12 +665,13 @@ func (ct *ConnectivityTest) deploy(ctx context.Context) error {
 			testConnDisruptClientDeployment := newDeployment(deploymentParameters{
 				Name:     testConnDisruptClientDeploymentName,
 				Kind:     KindTestConnDisrupt,
-				Image:    "quay.io/cilium/test-connection-disruption:v0.0.4",
+				Image:    "quay.io/cilium/test-connection-disruption:v0.0.5",
 				Replicas: 5,
 				Labels:   map[string]string{"app": "test-conn-disrupt-client"},
 				Port:     8000,
 				Command: []string{
 					"tcd-client",
+					"--dispatch-interval", "10", // 10ms
```

Comment:
unrelated to this pr, but in general it's better to use duration flags so that you can simply say `"--dispatch-interval", "10ms"`

Reply:
:+1: will do it as a follow-up